### ISSUE TITLE
GStreamer Cache doesn't regenerate after every reboot on H9

### ISSUE THREAD

**Issue by fredldotme** [ubports/ubuntu-touch#1904]
- Device: Pixel 3a (sargo}
- Channel: RC
- Build: 2021-W52

### Steps to reproduce

- Boot phone
- Record video
- Play back video

### Expected behavior

The video saved successfully.

### Actual behavior

The video got corrupted during record and doesn't play.

### Logfiles and additional information

The problem lies in the Pixel 3a port having its codec strategy changed: https://github.com/fredldotme/android_device_google_bonito/commit/9bb8f27777f7019812497885995e8d58d5f57a23

It worked during testing, probably because the GStreamre registry cleanup job triggered for me but not for every other user:
https://gitlab.com/ubports/core/rootfs-builder-debos/-/commit/adf1bae7661690974745759faaa4c6488282d336

This should be fixed by starting the cleanup job also before media-hub, which it might have raced with.

**Comment by fredldotme** [comment]
This issue seems to be weirder than initially I anticipated. I had to revert the patch that caused the problem on the port-side to get everything back to normal.
There seems to be some race condition present between Android init setting the property to disable CCodec, and the actual use of the media-related interfaces to interact with the OMX service.
Not every bootup exposes this problem, but some do.

**Comment by Flohack74** [comment]
So do we need to open a new issue?

**Comment by fredldotme** [comment]
I don't think so, and I think we can close this one too as the initial analysis was wrong.
The Pixel 3a is the only Halium 9.0 device with a split personality wrt media codecs (Codec 1.0 AND 2.0 in one vendor partition).
It's a non-issue as long as we keep the Codec 2.0 the default as it was pre-OTA-21. Therefore I'll close the issue.

**Comment by SevralT** [comment]
For me (H9) videos always saved successfully. Recorded video rn and file is no corrupted.

**Comment by fredldotme** [comment]
Reopening for now since some uses report failures in the forums still.

**Comment by matzgewinn** [comment]
Device is OnePlus 5, RC channel, 2022-W45. Video recording and saving works.

**Comment by ilyakooo0** [comment]
Can not reproduce the issue on the latest RC on the Pixel 3a
